Formula/octant.rb

```diff
@@ -35,6 +35,18 @@ def install
 
     Language::Node.setup_npm_environment
 
+    # Work around build error: "npm ERR! Invalid Version: ^3.0.8"
+    # Issue is due to `npm-force-resolutions` not working with
+    # npm>=8.6, which is used in node>=16 formulae.
+    #
+    # PR ref: https://github.com/vmware-tanzu/octant/pull/3311
+    # Issue ref: https://github.com/vmware-tanzu/octant/issues/3329
+    # Issue ref: https://github.com/rogeriochaves/npm-force-resolutions/issues/56
+    ENV.prepend_path "PATH", Formula["node@14"].opt_bin
+    cd "web" do
+      system "npm", "install", *Language::Node.local_npm_install_args
+    end
```

Comment:
Alternative 1: Install `npm@8.5.5` fetching via `npm` (allows building with `node`):
```rb
    system "npm", "install", *Language::Node.local_npm_install_args, "npm@8.5.5"
    ENV.prepend_path "PATH", buildpath/"node_modules/.bin"
```

Alternative 2: Similar to 1 but use a resource for `npm-8.5.5`

---

- In favor of current PR: `npm` 8.5.5 may have bugs while `node@14` (and thus bundled `npm`) will at least be maintained for a bit longer.
- In favor of alternatives: `node@14` is EOL in 4 months.